.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       # Retrieve changes in SuperBuild-related files relative to the 'nightly-main' branch
       - name: "Detect SuperBuild Changes"
-        uses: tj-actions/changed-files@6f67ee9ac810f0192ea7b3d2086406f97847bcf9 # v46.0.3
+        uses: tj-actions/changed-files@9934ab3fdf63239da75d9e0fbd339c48620c72c4 # v46.0.3
```

Comment:
Looks like we are not getting consistent "string comment" updates following new releases. See https://github.com/tj-actions/changed-files/tags

I would have expected the updated to be `ed68ef82c095e0d48ec87eccea555d944a631a4c # v46.0.5`

![image](https://github.com/user-attachments/assets/8c039d9e-189a-4d04-ab1c-0380a9b571df)

Reply:
@jamesobutler This may be related to https://github.com/Slicer/Slicer/pull/8312#issuecomment-2726572424

Reply:
I manually updated the comment recently (in https://github.com/Slicer/Slicer/pull/8327) after that security event to get the comment to match again. I could have put the wrong version comment, but I suspect the maintainer may still be manipulating the tags in some way after the fact that breaks the dependabot update.

I've pushed to get in back in sync again.